libcodechecker/libhandlers/parse.py

```diff
@@ -158,7 +158,8 @@ def arg_match(options):
     parser.set_defaults(func=__handle)
 
 
-def parse(f, context, metadata_dict, suppress_handler, skip_handler, steps):
+def parse(f, context, metadata_dict, suppress_handler, skip_handler,
+          processed_path_hashes, steps):
```

Comment:
I would prefer to extend the function arguments at the end if possible. It is really easy to mix up different type of arguments in python.